styles/hydrogen.less

```diff
@@ -43,7 +42,7 @@
                 line-height: 1.5em;
                 text-align: center;
                 box-shadow: 0 0 2px 0px rgba(0,0,0,0.5);
-                margin: 0;
+                margin: -50% 0 0 -50%;
```

Comment:
<img width="234" alt="bildschirmfoto 2016-03-30 um 23 49 42" src="https://cloud.githubusercontent.com/assets/13285808/14158746/2bbb2908-f6d2-11e5-866c-86ec144f7b7e.png">
Do you like this one better?